app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+
     <application
         android:name=".ui.main.HiltActivityClass"
```

Comment:
NarutoQuizApp olarak isimlendirebiliriz. `HiltActivityClass` ile sadece Hilt için eklemişiz gibi duruyor. App genelini kapsayan init işlemleri için burası kullanılabiliyor. Örneğin Firebase init işlemleri.